_data/sidebars/tutorial_sidebar.yml

```
@@ -49,6 +49,10 @@ entries:
     - title: Flow over heated plate steady state
       url: /tutorials-flow-over-heated-plate-steady-state.html
       output: web
+    
+    - title: Flow over heated plate with two meshes
```

Shorter:

```suggestion
    - title: Flow over heated plate (two meshes)
```

But only it we also apply the same style to the "steady state" and "nearest projection" tutorials, actually.

---

Probably best yes, "Flow over heated plate steady state" without parentheses is ugly.